packages/odyssey-storybook/src/components/odyssey-mui/PasswordField/PasswordField.stories.tsx

```diff
@@ -66,6 +66,9 @@ const storybookMeta: Meta<PasswordFieldProps> = {
     isOptional: {
       control: "boolean",
     },
+    hasShowPassword: {
+      control: "boolean",
+    },
```

Comment:
We should default this to `true` in the args table.

Reply:
it's already default to true within `PasswordField`, so do we benefit from also setting it in the stories?

Reply:
added after seeing the default value for `isOptional` is still added in the `args` table